src/main/java/propertyFiles/codeMeta20.properties

```diff
@@ -1,5 +1,6 @@
 metadatablock.name=codeMeta20
-metadatablock.displayName=Software Metadata (CodeMeta 2.0)
+metadatablock.displayName=Software Metadata (CodeMeta v2.0)
```

Comment:
This is a good fix, making it the same as what's in scripts/api/data/metadatablocks/codemeta.tsv